src/pages/Stream/Views/Explore/StaticLogTable.tsx

```diff
@@ -191,13 +200,20 @@ const ErrorView = (props: { message: string }) => {
 
 const LoadingView = () => {
 	return (
-		<Stack w="100%" align="center" h="100%" style={{ alignItems: 'center', justifyContent: 'center' }}>
-			<Loader variant="dots" />
+		<Stack
+			w="100%"
+			align="center"
+			h="100%"
+			style={{
+				alignItems: 'center',
+				justifyContent: 'center',
+			}}>
+			<Loader type="parseable" />
 		</Stack>
 	);
 };
 
-const Footer = (props: { loaded: boolean }) => {
+const Footer = (props: { loaded: boolean; isLoading: boolean; hasNoData: boolean }) => {
```

Comment:
Can you rename the props with more appropriate names ?

Reply:
ok